template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPageWithPageModel/.template.config/template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "ASP.NET"],
+  "name": "Razor Page with Page Model",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorPageWithPageModel",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorPageWithPageModel",
+  "shortName": "page",
```

Comment:
Ideas about a short name to distinguish this from pages (without pagemodel)??

We also have the option to make this a single item and have another parameter to distinguish.